frontend-web/webclient/app/UCloud/FilesApi.tsx

```diff
@@ -987,9 +1025,10 @@ export async function addFileSensitivityDialog(file: UFile, invokeCommand: Invok
     if (!isSensitivitySupported(file)) {
         dialogStore.addDialog(
             <>
-                <Heading.h2>Sensitive files not supported <Icon name="warning" color="errorMain" size="32" /></Heading.h2>
+                <Heading.h2>Sensitive files not supported <Icon name="warning" color="errorMain"
+                                                                size="32"/></Heading.h2>
```

Comment:
This is a nitpicking comment, sorry about that.

```suggestion
                <Heading.h2>
                    Sensitive files not supported <Icon name="warning" color="errorMain" size="32"/>
                </Heading.h2>
```